integrationTests/package.json

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
How does this get executed by circleci?

Reply:
I noted yesterday that this needs to be addressed, will talk with @kalagarraj to enable this.

Reply:
It is enabled in my PR, you can merge after it is approved